src/main/resources/messages/LibertyBundles.properties

```diff
@@ -39,7 +39,7 @@ action.io.openliberty.tools.intellij.actions.RemoveLibertyProjectAction.text=Lib
 action.io.openliberty.tools.intellij.actions.RemoveLibertyProjectAction.description=Remove the selected Liberty project from the tool window
 
 # Messages for Liberty actions and pop-up dialog
-no.liberty.projects.detected=No Liberty Maven or Liberty Gradle projects detected in this workspace.
+no.liberty.projects.detected=\n 1. If no projects are open in the Intellij Explorer, open or create a Liberty project using the File menu. \n 2. If there is one or more existing Maven or Gradle projects open in the Intellij Explorer, try one of the following: \n \n \t- Configure the Liberty build plugin in the build file of an existing Maven or Gradle project.\n \t- Add a server.xml file to an existing Maven or Gradle project at 'src/main/liberty/config'.\n \t- Manually add an existing Maven or Gradle project to the Liberty Dashboard through the Intellij Explorer command palette action 'Liberty:'Add project to Liberty Dashboard'
```

Comment:
```suggestion
no.liberty.projects.detected=\n 1. If no projects are open in the Project Tool Window, open or create a Liberty project using the File menu. \n 2. If there is one or more existing Maven or Gradle projects open in the Project Tool Window, try one of the following: \n \n \t- Configure the Liberty build plugin in the build file of an existing Maven or Gradle project.\n \t- Add a server.xml file to an existing Maven or Gradle project at 'src/main/liberty/config'.\n \t- Manually add an existing Maven or Gradle project to the Liberty Tool Window using the 'Liberty: Add project to the tool window' action through the Search Everywhere window.
```

Reply:
Please also make sure to update the spacing based on what was discussed during the demo. I believe the tabs should be removed and there should be an extra new line between (1) and (2).

Reply:
<img width="342" alt="Screenshot 2023-11-24 at 7 43 53 PM" src="https://github.com/OpenLiberty/liberty-tools-intellij/assets/133862121/353d92f5-8fd3-4c6e-984f-2a7b4d4323f8">

Addressed the above review comment.

Reply:
@Rahul-Biju you addressed the spacing issue, but you did not change the wording from the first review comment: https://github.com/OpenLiberty/liberty-tools-intellij/pull/584#discussion_r1403932384

Reply:
@TrevCraw
 Word changes : Done.

Reply:
@Rahul-Biju The updates to the message look good, but you did not update the message in the corresponding test case